Resolution of Issue#3010 (#3191)
MicrosoftDocs/windows-itpro-docs: windows/deployment/windows-autopilot/windows-autopilot-reset-remote.md

>[!NOTE]
>The Autopilot Reset option will not be enabled in Microsoft Intune for devices not running Windows 10 build 17672 or higher.

>[!IMPORTANT]

> >To use the Autopilot Reset option your device must be enrolled in a Autopilot Device.

Reviewer: To use the Autopilot Reset **option,** your device must be enrolled in **an** Autopilot **device**.

Kindly do changes in bold (insert comma, change a to an, and change upper case D to lower case d in device).

Reviewer 2: Practical way of showing it: use the "Insert a suggestion `<Ctrl+G>`" feature:
```suggestion
>To use the Autopilot Reset option, your device must be enrolled in an Autopilot device.
```

Reviewer 3: @illfated Thank you, will investigate this as a way to improve our process.

> >To use the Autopilot Reset (preview) option, your device **must be** reset using Autopilot *(either using Fresh Reset or manually sysprep the device)*, in any other case the feature will stay **grayed out**.

Reviewer 2: I must admit that I prefer @Neckross' original phrasing:
```suggestion
>The feature for Autopilot Reset (preview) will stay grayed out, **unless** you reset the device using Autopilot (either using Fresh Reset or manually sysprep the device).
```
My suggestion is not the blueprint or final answer. The MS Doc team members (or associates) might have a better suggestion for this added note.

Reviewer 4: @j0rt3g4 - Kindly check on the suggestion from @illfated and make the necessary changes. Thank you.
